libdrgn/stack_trace.c

```diff
@@ -460,7 +465,7 @@ static bool drgn_thread_set_initial_registers(Dwfl_Thread *thread,
 	err = linux_helper_task_state_to_char(&obj, &state);
 	if (err)
 		goto out;
-	if (state == 'R') {
+	if (state == 'R' && (prog->flags & DRGN_PROGRAM_IS_LIVE)) {
```

Comment:
I'd like to understand this case more, so would you mind leaving it for a followup PR?

Reply:
Sounds good I took that part out for now